default.nix

```diff
@@ -0,0 +1,20 @@
+# default.nix
+# with import <nixpkgs> {};
+# with import "github:NixOS/nixpkgs" {};
+let
+  nixpkgs = builtins.fetchTarball {
+    url    = "https://github.com/NixOS/nixpkgs/archive/0ba2543f8c855d7be8e90ef6c8dc89c1617e8a08.tar.gz";
+    sha256 = "14ann7vz7qgfrw39ji1s19n1p0likyf2ag8h7rh8iwp3iv5lmprl";
+  };
+  pkgs = import nixpkgs {};
+in pkgs.stdenv.mkDerivation {
+    name = "triphut-drv";
+    src = "./";
+    buildInputs = 
+      [ pkgs.pkg-config
+        pkgs.zlib
+        pkgs.secp256k1
+        pkgs.libsodium
+        pkgs.systemd
```

Comment:
Systemd won't work on any unix based systems other than Linux since `systemd` is Linux based

```error:
       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       … while evaluating derivation 'triphut-drv'
         whose name attribute is located at /nix/store/0nxa86wcgdpv3aqv4f5pyjlkaxg4fhwa-source/pkgs/stdenv/generic/make-derivation.nix:278:7

       … while evaluating attribute 'buildInputs' of derivation 'triphut-drv'

         at /nix/store/0nxa86wcgdpv3aqv4f5pyjlkaxg4fhwa-source/pkgs/stdenv/generic/make-derivation.nix:322:7:

          321|       depsHostHost                = lib.elemAt (lib.elemAt dependencies 1) 0;
          322|       buildInputs                 = lib.elemAt (lib.elemAt dependencies 1) 1;
             |       ^
          323|       depsTargetTarget            = lib.elemAt (lib.elemAt dependencies 2) 0;

       error: Package ‘systemd-250.4’ in /nix/store/0nxa86wcgdpv3aqv4f5pyjlkaxg4fhwa-source/pkgs/os-specific/linux/systemd/default.nix:700 is not supported on ‘aarch64-darwin’, refusing to evaluate.

       a) To temporarily allow packages that are unsupported for this system, you can use an environment variable
          for a single invocation of the nix tools.

            $ export NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1

        Note: For `nix shell`, `nix build`, `nix develop` or any other Nix 2.4+
        (Flake) command, `--impure` must be passed in order to read this
        environment variable.

       b) For `nixos-rebuild` you can set
         { nixpkgs.config.allowUnsupportedSystem = true; }
       in configuration.nix to override this.

       c) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
         { allowUnsupportedSystem = true; }
       to ~/.config/nixpkgs/config.nix.```

Reply:
Ah yes, this was really more just for myself to get things going locally before adding a proper flake setup. I added an issue for that #6 - which I'll take care of probably early next week. 

Maybe it makes sense to remove this `default.nix` from here in the meantime?

Reply:
Sure, that would work since only you're using it.